images/dvcr/werf.inc.yaml

```
@@ -18,30 +18,37 @@ imageSpec:
     user: 64535
 ---
 {{- $version := "2.8.3" }}
+{{- $gitRepoUrl := "distribution/distribution.git" }}
 
 image: {{ $.ImageName }}-builder
 final: false
-fromImage: base-alt-p10
+fromImage: builder/alt
```

```suggestion
fromImage: builder/golang-bookworm-1.23
```

go builder already has git and openssh. No need to add "apt packages proxy".

See https://github.com/deckhouse/virtualization/pull/1039/files#diff-37b04bc8fd51253cf7d71e6bf33a9bf5161492bfcb55d24867b20387fcfca9eb